lib/season_stat.rb

```diff
@@ -58,7 +57,7 @@ def get_team_info(season)
          season_win_percent: team_win_percentage(team.team_id, 'Regular Season', season),
          postseason_win_percent: team_win_percentage(team.team_id, 'Postseason', season)
 }
-      @team_info = team_hash
+      team_hash
```

Comment:
refactored @team_info out of season class, it seemed unnecessary to store a hash with season variables when i could just call the method and take the return value

Reply:
i agree here!